Lib/sqlite3/test/dbapi.py

```diff
@@ -170,10 +171,16 @@ def test_in_transaction_ro(self):
         with self.assertRaises(AttributeError):
             self.cx.in_transaction = True
 
+
+class OpenTests(unittest.TestCase):
+    def tearDown(self):
+        # bpo-42213: ensure that TESTFN is closed before unlinking
+        gc_collect()
```

Comment:
I am against a PR which calls gc.collect() to ensure that a resource is released. It doesn't look reliable at all. It sounds like a way to hide a real bug, and it will likely bite us later.

Reply:
I agree; I'll revert the workaround.